docs/future/vite.md

```diff
@@ -723,6 +723,8 @@ check out our [discussion on Hot Module Replacement][hmr] to learn more about th
 
 #### ESM / CJS
 
+For a walkthrough of fixing an example bug checkout [🎥 How to Fix CJS/ESM Bugs in Remix][how-fix-cjs-esm].
+
 Vite supports both ESM and CJS dependencies, but sometimes you might still run into issues with ESM / CJS interop.
 Usually, this is because a dependency is not properly configured to support ESM.
 And we don't blame them, its [really tricky to support both ESM and CJS properly][modernizing-packages-to-esm].
```

Comment:
I feel like we need the context of the issue before the link—what do you think?

```suggestion
Vite supports both ESM and CJS dependencies, but sometimes you might still run into issues with ESM / CJS interop.
Usually, this is because a dependency is not properly configured to support ESM.
And we don't blame them, its [really tricky to support both ESM and CJS properly][modernizing-packages-to-esm].

For a walkthrough of fixing an example bug, check out [🎥 How to Fix CJS/ESM Bugs in Remix][how-fix-cjs-esm].
```